src/pages/worklist/DoiRequests.tsx

```diff
@@ -23,7 +22,7 @@ const DoiRequests: FC = () => {
   ) : (
     <>
       {registrationsWithPendingDoiRequest.map((registration) => (
-        <DoiRequestAccordion key={registration.identifier} registration={registration} />
+        <DoiRequestAccordion key={registration.identifier} identifier={registration.identifier} />
```

Comment:
Tror godkjente DoiRequests fortsatt skal dukke opp i lista, bare med status "Ferdig", så dette kan jo bli et problem etter hvert når det blir 100+ DoiRequests siden hver Accordion da vil hente hver sin Registration hver gang.  Litt bakvendt å først hente alle DoiRequester her, for så å hente alle en gang til i Accordion, pluss at det kan bli ganske mange requests

Men det hadde sikkert blitt problemer om vi refetcha alle DoiRequests for hver melding også...

Vi kunne utvida `useFetchRegistration` til å ta inn en `initRegistration`, slik at den ikke måtte hente på nytt på onmount hvis den allerede har. Men da må vi evt fortsatt hente alle DoiRequests igjen når man lukker en Accordion for at den ikke skal få gammel data om den åpnes igjen senere.

Ser egentlig bare problemer og ingen gode løsninger her 😅

Reply:
ja, jeg tenkte også på dette, og prøvde alt det du foreslo over der, men det ble også unødvendig komplisert, og alt ble refetchet hele tiden, så accordion-en lukket seg hver gang man sendte en melding. Jeg synes egentlig det er best at komponentene er mest mulig selvstendig, så jeg foretrekker at accordionen henter data selv nå. Men ser poenget med ferdige doi-requests i fremtiden ja. Men kanskje ta det når det kommer?

Reply:
går evt an å prop-drille alt som ikke har med doi å gjøre da. og kun hente doi-meldinger på nytt? ser ikke for meg at man skal kunne sende meldinger på en ferdigbehandlet doi-request? men tenker egentlig uansett at vi kan ta dette når det kommer da

Reply:
ja, vi kan avvente